### DISCUSSION TITLE
Track0 signal asserted after seek to cylinder - what does is exactly mean?

### DISCUSSION THREAD

**Discussion by cwndsrfr** [keirf/greaseweazle#327] [Q&A]
Hello,

First off, just wanted to give my thanks to creating this wonderful device.  I am absolutely delighted to be able to read stuff off my old disks after 35 years.  I have been troubleshooting one of my old drives. It is a Fujitsu M2551A 08 360KB DS DD 5.25 Floppy drive.

When I try and read an image off this drive in Greaseweazle, i get the error "Track0 signal asserted after seek to cylinder 1".  This happens after T0.1.

In ImageDisk, I have confirmed that I am able to read both sides of the disk (Head 0 and 1) in all tracks in the alignment mode. I have realigned the heads and the track 0 stop sensor a few times but keep getting the same issue. Also, when I try and format a 360kb disk in DOS 3.2, it only formats 1 side (Head 0), which results in a 160kb formatted disk. So clearly there is an issue with this drive but wanted to understand the error that greaseweazle is throwing since ImageDisk doesn't seem to think there is an issue.

What causes the drive to assert track 0 when seeking to next track?  Any help or insight into this error is appreciated.

I have 2 other 5.25 drives that have been working flawlessly in greaseweazle: A Panasonic JU-47504 1,2 MB 5.25 drive and a Teac FD-55BV 360Kb 5.25 drive.

Thanks

**Comment by keirf** [comment]
It might need a slower step speed. Check out `gw delays` to observe current step delay and adjust via `gw delays --step`. You could try a much bigger number like 30000 microseconds to start with.

**Comment by cwndsrfr** [comment]
Thanks for the suggestion. I tried setting a delay but unfortunately it keeps doing the same thing. As I mentioned, there is clearly an issue with the drive but I believe the heads are ok. There must be an issue with the circuitry and am trying to pinpoint the cause. I might start replacing some caps on the board.

**Reply by keirf** [reply]
Greaseweazle is deliberately fussy about checking that track 0 is only asserted at track (actually cylinder) 0. Perhaps the track 0 sensor is covering more than one track? Do you hear the drive heads step with gw (to cylinder 1) when the error occurs.
